src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs

```diff
@@ -237,7 +237,7 @@ public float GetSingle()
         /// <seealso cref="TokenType" />
         /// </exception>
         /// <exception cref="FormatException">
-        /// Thrown if the JSON token value represents a number less than <see cref="double.MinValue"/> or greater 
+        /// On .NET Framework, thrown if the JSON token value represents a number less than <see cref="double.MinValue"/> or greater 
```

Comment:
What should we do about https://github.com/dotnet/corefx/issues/33997#issuecomment-447167407
> The real consequence of option 2 is that we should get rid of TryGetDouble, since it can never return false. 

Looks like we can't really get rid of `TryGetDouble`, even though, on .NET Core, it will not return false.

Reply:
Right, it should exist for consistency and since it can fail on full framework.

Reply:
It can also return `false` for any string which isn't a number sequence, if that is ever extended to be allowed in the future.

Reply:
My "we should delete it" really only applied when the API was netcoreapp30+  Now that it's netstandard20 the method has value.